web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -140,7 +146,57 @@ export default defineComponent({
       }
     });
 
-    onMounted(() => {
+    const page = ref(Number(ctx.root.$route.query.pos) || 1);
+    const pages = ref(1);
+    const nextDandiset : Ref<any[]> = ref([]);
+
+    const djangoDandisetRequest: Ref<Paginated<Dandiset> | null> = ref(null);
+    async function fetchNextPage() {
+      const sortOption = Number(ctx.root.$route.query.sortOption) || 0;
+      const sortDir = Number(ctx.root.$route.query.sortDir || -1);
+      const sortField = sortingOptions[sortOption].djangoField;
+      const ordering = ((sortDir === -1) ? '-' : '') + sortField;
+      const response = await dandiRest.dandisets({
+        page: page.value,
+        page_size: 1,
+        ordering,
+        search: ctx.root.$route.query.search,
+        draft: true,
+        empty: true,
+      });
+      djangoDandisetRequest.value = response.data;
+
+      pages.value = (djangoDandisetRequest.value?.count) ? djangoDandisetRequest.value?.count : 1;
+      nextDandiset.value = djangoDandisetRequest.value?.results.map((dandiset) => ({
+        ...(dandiset.most_recent_published_version || dandiset.draft_version),
+        contact_person: dandiset.contact_person,
+        identifier: dandiset.identifier,
+      }));
```

Comment:
Related to the above comment


```suggestion
      pages.value = response.data?.count || 1;
      nextDandiset.value = response.data?.results.map((dandiset) => ({
        ...(dandiset.most_recent_published_version || dandiset.draft_version),
        contact_person: dandiset.contact_person,
        identifier: dandiset.identifier,
      }));
```